config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm.yaml

```diff
@@ -0,0 +1,488 @@
+periodics:
+- interval: 2h
+  name: ci-kubernetes-e2e-kubeadm-gce
+  agent: kubernetes
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180730-c8036c3a7
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --timeout=320
+      - --upload=gs://kubernetes-jenkins/logs
+      - --scenario=kubernetes_e2e
+      - --
+      # TODO(bentheelder): --kubernetes-anywhere-kubernetes-version should really be ci/latest-bazel but we need changes to kubeadm and possibly kubernetes-anywhere first
+      - --cluster=
+      - --deployment=kubernetes-anywhere
+      - --extract=ci/latest-bazel
+      - --gcp-zone=us-central1-f
+      - --kubeadm=ci
+      - --kubernetes-anywhere-kubelet-ci-version=latest-bazel
+      - --kubernetes-anywhere-kubernetes-version=ci-cross/latest
+      - --provider=kubernetes-anywhere
+      - --test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\] --minStartupPods=8
+      - --timeout=300m
+
+
+- name: ci-kubernetes-e2e-kubeadm-gce-1-10
+  interval: 6h
+  agent: kubernetes
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/e2e-kubeadm:v20180730-c8036c3a7
+      args:
+      - --repo=k8s.io/kubernetes=release-1.10
+      - --timeout=320
+      - --upload=gs://kubernetes-jenkins/logs
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=
+      - --deployment=kubernetes-anywhere
+      - --extract=ci/latest-bazel-1.10
+      - --gcp-zone=us-central1-f
+      - --kubeadm=ci
+      - --kubernetes-anywhere-kubelet-ci-version=latest-bazel-1.10
+      - --kubernetes-anywhere-kubernetes-version=ci/latest-1.10
+      - --provider=kubernetes-anywhere
+      - --test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\]
+      - --timeout=300m
+
+# TODO(bentheelder): switch ci-kubernetes-bazel-build to 100% periodic,
```

Comment:
hmm this comment seems stale. :grimacing:

Reply:
removed :-)